app/__init__.py

```diff
@@ -15,11 +15,11 @@
 def create_app(config_name):
     application = Flask(__name__)
     application.wsgi_app = ProxyFix(application.wsgi_app)
-    application.config.from_object(config[config_name])
-
-    config[config_name].init_app(application)
+    application.config['DM_ENVIRONMENT'] = config_name
```

Comment:
It's not new, it's being used in `application.py`. All I've done here is add it to the app config. I figured it might be a useful thing to be able to reference.